enterprise/utils.py

```diff
@@ -1431,10 +1477,16 @@ def enroll_users_in_course(
                     course_id,
                     role=MANUAL_ENROLLMENT_ROLE,
                 )
+
+            if license_uuids:
+                # create licensed enrollment if a subscription license is supplied
+                enterprise_customer_user = get_enterprise_customer_user(user.id, enterprise_customer.uuid)
+                get_create_ent_enrollment(course_id, enterprise_customer_user, license_uuids.get(user).get(course_id))
```

Comment:
will this work? isn't the key to license_uuids supposed to be the email address?

Reply:
`license_uuids.get(<user>).get(<course_id>)`
should be getting the license uuid from the structure:
```
                license_info: {
                    <user email>: {
                        <course_id>: <license UUID that I'm trying to fetch>                    },
                }
```
no?

Reply:
yup what I had meant was is `user` above the email address or the entire user object? just maybe good to confirm.